packages/example-corvair/package.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
I thought we couldn't use the name `corvair` because it's a registered trademark?